frontend/packages/helm-plugin/src/components/details-page/resources/HelmReleaseResourcesRow.tsx

```diff
@@ -20,10 +20,10 @@ export const HelmReleaseResourceStatus: React.FC<HelmReleaseResourceStatusProps>
   const kind = referenceFor(resource);
   return resource.status?.replicas ? (
     <Link
-      to={`${resourcePath(kind, resource.metadata.name, resource.metadata.namespace)}/pods`}
+      to={`${resourcePath(kind, resource.metadata?.name, resource.metadata?.namespace)}/pods`}
```

Comment:
The link will be broken if these values aren't set. I'd almost rather it be a runtime error. We need to handle those cases if they can happen.

I wonder if we need another type like `K8sResourceInstance` or something that indicates it's a "real" instance of a resource and these fields are always set.